### PR TITLE
February news

### PR DESCRIPTION
**Pull request by maelle**
TODO 

- [x] don't list existing packages as new packages :-)
- [x] Add links to calls for contributions

### CODE REVIEW

File: content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,652 @@
...
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### rOpenSci 2023 Code of Conduct Transparency Report
+
+Read the [report](/blog/2024/01/12/transparency2023/).
```
Comment (yabellini) [inline content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd:48]:
```suggestion

Transparency reports are intended to help the community understand what kind of Code of Conduct incidents we receive reports about annually and how the Code of Conduct team responds, always preserving the privacy of the people who experience or report incidents.

Read the [report](/blog/2024/01/12/transparency2023/).
```

File: content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,655 @@
...
+
+## rOpenSci HQ
+
+### rOpenSci 2023 Code of Conduct Transparency Report
+
+
+Transparency reports are intended to help the community understand what kind of Code of Conduct incidents we receive reports about annually and how the Code of Conduct team responds, always preserving the privacy of the people who experience or report incidents.
+
+Read the [report](/blog/2024/01/12/transparency2023/).
+
```
Comment (yabellini) [inline content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd:52]:
```suggestion

### rOpenSci Champions Program 

We are proud to welcome our second cohort of Champions!  Learn about them and the projects they will develop while participating in the rOpenSci Champions Program.

Read the [blog post](/blog/2024/02/15/champions-program-champions-2024/)
  

```

**Comment by jeroen** [comment]
If it is still in time, we can mention the new fluidsynth package: https://github.com/ropensci/roweb3/pull/734

**Comment by jeroen** [comment]
Two more small r-universe updates:

 - Thanks to [contributions from Hiroaki Yutani](https://github.com/r-wasm/webr/pull/358#issuecomment-1954019290) r-universe now has experimental support for compiling packages with rust code into wasm.
 - In preparation for the next major R release, we started building MacOS binaries for 4.4 and will soon drop 4.2 binaries.

File: content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,662 @@
...
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, March 5th, 9:00 Australia Western (1:00 UTC)](/events/coworking-2024-03/), Dates, Times and Timezones in R. With cohost [Steffi LaZerte](/author/steff-lazerte/) and [Alex Koiter](/author/alex-koiter/).
+    - Explore resources for working with dates, times, and timezones in R
+    - Work on a project dealing with dates and times
+    - Ask questions or troubleshoot your timezone problems with the cohost and other attendees.
+
```
Comment (steffilazerte) [inline content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd:74]: Worth including? I might have something by Thursday, but not guarantee.
```suggestion
    
- Tuesday, April 2nd, 14:00 Europe Central (13:00 UTC) Theme and Cohost TBA
    
```

File: content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,662 @@
...
+Read [all about coworking](/blog/2023/06/21/coworking/)!
+
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, March 5th, 9:00 Australia Western (1:00 UTC)](/events/coworking-2024-03/), Dates, Times and Timezones in R. With cohost [Steffi LaZerte](/author/steff-lazerte/) and [Alex Koiter](/author/alex-koiter/).
```
Comment (steffilazerte) [inline content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd:70]:
```suggestion
- [Tuesday, March 5th, 9:00 Australia Western (1:00 UTC)](/events/coworking-2024-03/), Dates, Times and Timezones in R. With cohost [Steffi LaZerte](/author/steffi-lazerte/) and [Alex Koiter](/author/alex-koiter/).
```

File: content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,662 @@
+---
+title: 'rOpenSci News Digest, February 2024'
+author: 'The rOpenSci Team'
+date: '2024-02-23'
+tags:
+  - newsletter
+slug: news-january-2024
```
Comment (steffilazerte) [inline content/blog/2024-02-23-ropensci-news-digest-february-2024/index.Rmd:7]: Should this be February?
```suggestion
slug: news-february-2024
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle,
Very few comments from me. I might have a coworking for April by Thursday, but maybe (probably) not. I included the placeholder for now at least. 
Thanks!

**Comment by maelle** [comment]
Thanks @steffilazerte!!

**Pull request merged**